org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -141,4 +141,116 @@
       }
     }
   }
+
+  &.page-civicrm-vacancy-publiclisting table {
+    @extend %crm-table;
```

Comment:
Why this styles are applied only on this particular table? I presume `%crm-table` contains the style that all "old civi" tables should have? If so, do you think this styles here should go in that placeholder? Or is this table an exception?